modal/functions.py

```diff
@@ -563,7 +569,7 @@ class _Function(_Object, type_prefix="fu"):
     # TODO: more type annotations
     _info: Optional[FunctionInfo]
     _all_mounts: Collection[_Mount]
-    _stub: "modal.stub._Stub"
+    _stub: Optional["modal.stub._Stub"] = None  # not set in case of external lookups
```

Comment:
The _stub could actually be unset before, now it's more explicitly set to None in those cases (mainly Functions received from `Function.lookup()`)